src/utils/signTypedData.ts

```diff
@@ -1,5 +1,5 @@
 // https://github.com/enzymefinance/protocol/blob/c9621dd5f8234bd45126772fc626252a38d46eee/packages/ethers/src/utils/signTypedData.ts
-import { JsonRpcProvider } from '@ethersproject/providers';
+import { JsonRpcProvider, JsonRpcSigner } from '@ethersproject/providers';
```

Comment:
duplicate import?

Reply:
It was to give an explicit type to `signer?`

We could also just remove `signer?` and do `provider: JsonRpcProvider | JsonRpcSigner`

Reply:
oh sorry misread, my bad.